### PR TITLE
Add moment.js to extension date

### LINKED ISSUE #404: Format extension dates with moment.js
- [x] Dates like this on the assessment page should be formatted using the same moment.js formatting as other dates.

screen shot 2015-03-18 at 4 35 21 am

### PR DESCRIPTION
**Pull request by dlbucci**
Fixes #404

### CODE REVIEW

File: app/views/assessments/show.html.erb [hunk 1]
```diff
@@ -112,7 +112,7 @@
   You have a <%=@extension.days %> day extension on this assessment
   </i>
   <font size=-1>
-  (Due date: <%= due_at_display(@assessment.aud_for(@effectiveCud.id).due_at) %>)
+    (Due date: <span class="moment-date-time"><%= due_at_display(@assessment.aud_for(@effectiveCud.id).due_at) %></span>)
   </font>
```
Comment (jez) [inline app/views/assessments/show.html.erb]: What in the world do we have a font tag for?

Comment (dlbucci) [inline app/views/assessments/show.html.erb]: I don't know! It's as if some of our HTML is sub-par!

**Comment by icanberk** [comment]
This is a hotfix but merged into `hotfix_extension_moment_date`? Is it actually a hotfix?

**Comment by dlbucci** [comment]
What?  Are you asking if it belongs in master?  Cause there's no urgency, I meant for it to be in develop, but it is a bug fix.  I guess fix might have been more appropriate.

**Comment by icanberk** [comment]
Yeah, we're supposed to use hotfix_ for urgent things that needs to go into master, but no probs.

**Pull request merged**